### PR TITLE
Replace deprecated instagram legacy API by instagram basic display API

### PR DESCRIPTION
**Pull request by Thoomyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,21 @@ class CustomPlugin extends Plugin
 }
 ```
 
+## Nwtworks
```
Comment (moduleon) [inline README.md]: Typo: Networks

File: blueprints.yaml [hunk 2]
```diff
@@ -83,16 +77,22 @@ form:
...
               type: text
-              label: Username
+              label: User ID
+            .access_token:
+              type: text
+              label: Access token
+            .avatar:
+              type: file
+              label: Avatar
+              help: Fallback image if no user image aviable by api
```
Comment (moduleon) [inline blueprints.yaml]: Typo: available I guess, not aviable

**Review by moduleon — COMMENTED** [review]
Hi Thomas, thank you for this interesting PR, I test it more deeply as soon as possible. Great Job, I would like you to add a section in the README, for Contributors, and add your name in it, if it's okay for you.

**Comment by Thoomyy** [comment]
Hi Kévin

I am not pressed ;)

That would be nice
Thx

Greeting
Thomas

Am Mi., 24. Juni 2020 um 19:42 Uhr schrieb Kévin Marcachi <
notifications@github.com>:

File: README.md (hunk 1 above)
Comment (Thoomyy) [reply README.md]: thx - resolved

File: blueprints.yaml (hunk 2 above)
Comment (Thoomyy) [reply blueprints.yaml]: thx - resolved

**Pull request closed without merging**